pages/mesosphere/dcos/services/hdfs/2.6.0-3.2.0/operations/configuration/index.md

```diff
@@ -48,7 +48,7 @@ In order for {{ model.techShortName }} to function correctly, you must perform s
     <td>/etc/sysctl.conf</td>
     <td>vm.swappiness</td>
     <td>0</td>
-    <td>If the OS swaps out the HDFS processes, they can fail to respond to RPC requests, resulting in the process being marked down by the cluster. This can be particularly troublesome for name nodes and journal nodes.</td>
+    <td>If the OS swaps out the HDFS processes, they can fail to respond to RPC requests, resulting in the process being marked `down` by the cluster. This can be particularly troublesome for name nodes and journal nodes.</td>
```

Comment:
Hey @ishanchandrajoshi actually can we add a section for using volume profiles here? Similar to what Cassandra is doing: https://docs.d2iq.com/mesosphere/dcos/services/cassandra/2.6.0-3.11.4/configuration/#using-volume-profiles

Reply:
There is  back tick around 'down'. But I don't know why is it not recognized when we run.

Reply:
Looks like code marks don't work inside markdown tables.  Let's change it to:
```suggestion
    <td>If the OS swaps out the HDFS processes, they can fail to respond to RPC requests, resulting in the process being marked DOWN by the cluster. This can be particularly troublesome for name nodes and journal nodes.</td>
```